.github/workflows/pkgdown.yaml

```diff
@@ -0,0 +1,49 @@
+on:
+  push:
+    branches:
+      - master
+      - docs
+
+
+name: pkgdown
+
+jobs:
+  pkgdown:
+    runs-on: macOS-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-r@master
+
+      - uses: r-lib/actions/setup-pandoc@master
+
+      - name: Query dependencies
+        run: |
+          install.packages('remotes')
+          saveRDS(remotes::dev_package_deps(dependencies = TRUE), ".github/depends.Rds", version = 2)
+          writeLines(sprintf("R-%i.%i", getRversion()$major, getRversion()$minor), ".github/R-version")
+        shell: Rscript {0}
+
+      - name: Cache R packages
```

Comment:
could we use this in test workflow too?

Reply:
Using names or specifying shells? Anyway, I'm afraid that I wouldn't be able do answer, I'm not too proficient with the workflow yaml code. This file was completely autogenerated by `usethis::use_pkgdown()` based on following a manual, starting at the [pkgdown web site](https://pkgdown.r-lib.org/). Note that you'll probably need to configure the GitHub pages yourself, to get the site published as https://smbache.github.io/import/ :

```

If you are using GitHub, the easiest way to make this your package 
website is to check into git, then go to settings for your repo and 
make sure that the GitHub pages source is set to 
“master branch /docs folder”. Be sure to update the URL on your 
github repository homepage so others can easily navigate to your new site.
```

Reply:
caching package installs for faster workflows :) something to play with some day.